app.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "Fewlines Connect Account",
+    "description": "Connect Account Management Application",
+    "scripts": {},
+    "env": {
+        "CONNECT_ACCOUNT_SESSION_SALT": {
+            "generator": "secret"
+        },
+        "CONNECT_APPLICATION_CLIENT_ID": "26bD6UeZZLFGmwuKNJoaJQ==",
+        "CONNECT_APPLICATION_SCOPES": "email phone openid",
+        "CONNECT_AUDIENCE": "connect-account",
+        "CONNECT_JWT_ALGORITHM": "RS256",
+        "CONNECT_MANAGEMENT_URL": "https://management.connect.review.fewlines.tech/graphql",
+        "CONNECT_OPEN_ID_CONFIGURATION_URL": "https://fewlines.connect.prod.fewlines.tech/.well-known/openid-configuration",
+        "CONNECT_PROVIDER_URL": "https://fewlines.connect.prod.fewlines.tech",
+        "CONNECT_REDIRECT_URI": "https://account.fewlines.tech/api/oauth/callback",
+        "DYNAMODB_ENDPOINT": "https://dynamodb.eu-west-3.amazonaws.com",
+        "DYNAMODB_REGION": "eu-west-3",
+        "DYNAMODB_TABLE_NAME": "fewlines-connect-account-review",
+        "IS_JWE_SIGNED": "true",
+        "NEXT_PUBLIC_SENTRY_DSN": "https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583",
+        "NEXT_TELEMETRY_DISABLED": "true"
+    },
+    "environments": {
+        "review": {
+          "env": {
+            "NODE_ENV": "development", 
+            "CONNECT_ACCOUNT_SESSION_SALT": {
+                "generator": "secret"
+            },
+            "CONNECT_APPLICATION_CLIENT_ID": "26bD6UeZZLFGmwuKNJoaJQ==",
+            "CONNECT_APPLICATION_SCOPES": "email phone openid",
+            "CONNECT_AUDIENCE": "connect-account",
+            "CONNECT_JWT_ALGORITHM": "RS256",
+            "CONNECT_MANAGEMENT_URL": "https://management.connect.review.fewlines.tech/graphql",
+            "CONNECT_OPEN_ID_CONFIGURATION_URL": "https://fewlines.connect.prod.fewlines.tech/.well-known/openid-configuration",
+            "CONNECT_PROVIDER_URL": "https://fewlines.connect.prod.fewlines.tech",
+            "CONNECT_REDIRECT_URI": "https://account.fewlines.tech/api/oauth/callback",
+            "DYNAMODB_ENDPOINT": "https://dynamodb.eu-west-3.amazonaws.com",
+            "DYNAMODB_REGION": "eu-west-3",
+            "DYNAMODB_TABLE_NAME": "fewlines-connect-account-review",
+            "IS_JWE_SIGNED": "true",
+            "NEXT_PUBLIC_SENTRY_DSN": "https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583",
+            "NEXT_TELEMETRY_DISABLED": "true"
```

Comment:
If we don't repeat those, won't it take the global `"env"` ones?

Reply:
Nope, I thought it would be that, but instead, it totally overrides the "base" ones, and sets just the ones that are beneath the current environment (`environments > review > env`).  cf -> https://devcenter.heroku.com/articles/app-json-schema#environments

Reply:
@tgautier do you have any insight to avoid this or is it not avoidable? 😇

Reply:
Even though I totally understand why this change was made, I think that we should keep in mind that the goal of having a CI/CD pipeline and review environments is to test the application behaviour in a production context before it actually reaches the real production.

For me, one of the key tenants of this is to make sure that the reviews, staging and production environments are as similar as possible to each other. I even think we should aim for them being strictly equivalent, except for some values (ENV variables, CPU/RAM resource allocation, etc...).

So, with this in mind, the `NODE_ENV=dev` is actually a big no-no for me, even though I totally understand why this change is suggested.

I would rather have us duplicate the code logic so that the script can be in vanilla JS and the code it was copied from can stay in TS in the actual app.

One of the nice benefits of this would also be that we no longer have to copy all those pesky ENV variables.